lib/generators/shopify_app/install/templates/content_security_policy.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+# Be sure to restart your server when you modify this file.
+
+# Define an application-wide content security policy
+# For further information see the following documentation
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+
+# Rails.application.configure do
```

Comment:
Are we sure we want to keep all of this stuff commented out in the generated file? I feel like since we require CSP to be set for all apps, we should be opinionated about actually applying the settings here. What do you think?

Reply:
I don't have a strong opinion on uncommenting the file, since the solution will be there in case anyone runs into issues. It might make sense to have the settings from the start, just in case folks try to develop a scenario that wouldn't work.